src/components/project/home-block.vue

```diff
@@ -230,12 +246,41 @@ export default {
       }
     }
 
-    .project-dataset-row .col-icon{
-      border-right-color: #b9005c;
+    .project-dataset-row {
+      .conflicts-count {
+        text-align: right;
+        padding-right: 8px;
```

Comment:
I think we could use `$padding-right-table-data` here. (Though isn't that the default?)